docs/how-payjoin-saves-exchanges.md

```diff
@@ -0,0 +1,117 @@
+---
+sidebar_position: 1
+---
+
+# How Can Payjoin Save 1/3 the Cost of Transacting?
+
+Payment batching is table stakes service for high-volume settlement services like exchanges and payment processors. But it has been limited to one party, the sender, combining multiple sends together. Ideally, multiple types of transfers could all be combined together. Imagine your deposit to an exchange was batched with withdrawals to others and lightning channel opens etc. This combination could save significant overhead to every transfer. And it provides an opportunity for much better privacy since, like bitcoin itself, only inputs and outputs are recorded but less information about which inputs and outputs are clustered would be revealed.
+
+Transactions compete to get included in blocks according to network fees they pay since each block is limited to a fixed weight. At a high level, each transaction weight pays some base costs (𝑏), per-input costs (𝑖) and per-output costs (𝑜). In reality not all inputs and outputs have equal cost but the principle can be explained assuming they are, and be backed up by examples of real transaction. For example, a simple exchange could have 5 btc in their treasury and sell 1 bitcoin each to Alice, bob, and Carol.
```

Comment:
```suggestion
Transactions compete to get included in blocks according to network fees they pay since each block is limited to a fixed weight. At a high level, each transaction weight pays some base costs (𝑏), per-input costs (𝑖) and per-output costs (𝑜). In reality not all inputs and outputs have equal cost but the principle can be explained assuming they are, and be backed up by examples of real transactions. For example, a simple exchange could have 5 btc in their treasury and sell 1 bitcoin each to Alice, bob, and Carol.
```